src/mongocrypt-marking.c

```diff
@@ -1436,7 +1438,8 @@ _mongocrypt_fle1_marking_to_ciphertext (_mongocrypt_key_broker_t *kb,
    }
 
    _mongocrypt_buffer_from_iter (&plaintext, &marking->v_iter);
-   ciphertext->data.len = _mongocrypt_calculate_ciphertext_len (plaintext.len);
+   ciphertext->data.len =
+      _mongocrypt_calculate_ciphertext_len (plaintext.len, status);
```

Comment:
Return an error if `_mongocrypt_fle2_calculate_ciphertext_len ` returns 0. Otherwise, an error status may be ignored by the caller.

Reply:
Change incoming.